### ISSUE TITLE
can't change feast assignment?

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#1198]
I am proofreading some chants but I can't seem to change the feast assignment to anything but "------"--and if it already is that, I can't change it at all! See below (this is https://cantusdatabase.org/edit-chants/672486?pk=673348&folio=012r)

![image](https://github.com/DDMAL/CantusDB/assets/67451875/11a420d3-0334-4374-9f6f-88bb6d665fee)

**Comment by jacobdgm** [comment]
When I visit https://cantusdatabase.org/edit-chants/672486?pk=673348&folio=012r, I don't see what's in your sceenshot, @annamorphism. This is what I see:

![Screenshot 2023-12-07 at 11 33 33 AM](https://github.com/DDMAL/CantusDB/assets/58090591/e7e2b814-d89f-47ab-816c-3b7296191e20)

The behavior of the Feast, Office, Genre and Differentia DB fields was updated on Production one week ago - they are all autocomplete fields now. Not sure when you took this screenshot, but if you took it within the past week, that would be very strange.

**Comment by annamorphism** [comment]
This was yesterday...but I'm not seeing it now either. I think I took this screenshot on my other computer, though, so maybe that could be a factor....??

**Comment by jacobdgm** [comment]
I'm not sure what may have been going on, but if this was indeed a bug, I can't replicate it. Going to close this issue - we can re-open it if this problem re-emerges.

**Comment by annamorphism** [comment]
Turns out I can replicate it easily on my other (older) computer. Seems to be some sort of browser issue--it works fine in Opera but not Firefox (which, on that computer, is a fairly old version...)

**Comment by annamorphism** [comment]
re-opening with low priority--I don't think it's *necessary* to be backwards-compatible for every academic with a coal-burning computer, but if there is an easy way of doing so that would be nice. (I do like editing on my desktop, even if it is from 2013...)

**Comment by jacobdgm** [comment]
Fair enough. I wonder whether django-autocomplete requires some feature that's only present in new browsers, and uses simple `<select>` elements as a fallback when that feature is missing.
